script.js

```diff
@@ -14,7 +16,412 @@ function removeMenu() {
+    technologies9: 'html',
+    liveLink: 'https://rachelwebdev.github.io/Portfolio-Project/',
+    sourceLink: 'https://github.com/Rachelwebdev/Portfolio-Project',
+    projectButton: 'See Project',
+    btnValue: 3,
+  },
+  {
+    projectHeading: 'Multi-Post Stories Gain+Glory',
+    projectName: 'Keeping track of hundreds of components',
+    projectDescription: `Lorem Ipsum is simply dummy text of the printing and typesetting
+          industry. Lorem Ipsum has been the industry's standard dummy text
+          ever since the 1500s, when an unknown printer took a galley of
+          type and scrambled it 1960s with the releaLorem Ipsum is simply
+          dummy text of the printing and typesetting industry. Lorem Ipsum
+          has been the industry's standard dummy text ever since the 1500s,
+          when an unknown printer took a galley of type and scrambled it
+          1960s with the release`,
+    projectImageMobile: './images/popup-mobile-img.svg',
+    projectImageDesktop: './images/popup-desktop-img.svg',
+    technologies1: 'Ruby on rails',
+    technologies2: 'css',
+    technologies3: 'JavaScript',
+    technologies4: 'Codekit',
+    technologies5: 'GitHub',
+    technologies6: 'Bootstrap',
+    technologies7: 'Terminal',
+    technologies8: 'Codepen',
+    technologies9: 'html',
+    liveLink: 'https://rachelwebdev.github.io/Portfolio-Project/',
+    sourceLink: 'https://github.com/Rachelwebdev/Portfolio-Project',
+    projectButton: 'See Project',
+    btnValue: 4,
+  },
+  {
+    projectHeading: 'Multi-Post Stories Gain+Glory',
+    projectName: 'Keeping track of hundreds of components',
+    projectDescription: `Lorem Ipsum is simply dummy text of the printing and typesetting
+          industry. Lorem Ipsum has been the industry's standard dummy text
+          ever since the 1500s, when an unknown printer took a galley of
+          type and scrambled it 1960s with the releaLorem Ipsum is simply
+          dummy text of the printing and typesetting industry. Lorem Ipsum
+          has been the industry's standard dummy text ever since the 1500s,
+          when an unknown printer took a galley of type and scrambled it
+          1960s with the release`,
+    projectImageMobile: './images/popup-mobile-img.svg',
+    projectImageDesktop: './images/popup-desktop-img.svg',
+    technologies1: 'Ruby on rails',
+    technologies2: 'css',
+    technologies3: 'JavaScript',
+    technologies4: 'Codekit',
+    technologies5: 'GitHub',
+    technologies6: 'Bootstrap',
+    technologies7: 'Terminal',
+    technologies8: 'Codepen',
+    technologies9: 'html',
+    liveLink: 'https://rachelwebdev.github.io/Portfolio-Project/',
+    sourceLink: 'https://github.com/Rachelwebdev/Portfolio-Project',
+    projectButton: 'See Project',
+    btnValue: 5,
+  },
+];
+
+const startingPoint = document.querySelector('.body');
+
+// -------------MOBILE POPUP---------------------------------
+function creatMobilePopup() {
```

Comment:
- [ ] I would also recommend you use the arrow function instead of the function keyword. The arrow function is ES6 which shows you are updated on the latest version usage of JS and it is of best practice. You can read it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)

Reply:
Thanks for this review.